pkg/server/endpoints/entryfetcher.go

```diff
@@ -49,16 +56,26 @@ func (a *AuthorizedEntryFetcherWithFullCache) FetchAuthorizedEntries(ctx context
 	return a.cache.GetAuthorizedEntries(agentID), nil
 }
 
+func (a *AuthorizedEntryFetcherWithFullCache) FetchAllCachedEntries() ([]*types.Entry, error) {
```

Comment:
is this only for debuggin right? if that is the case plz add a TODO with that just as a reminder